src/chrome/content/rules/McAfee.xml

```diff
@@ -2,91 +2,162 @@
 	For other Intel coverage, see Intel.xml.
 
 	Other McAfee rulesets:
-
+		- McAfee_Mobile_Security.xml
 		- McAfee-MX-Logic.xml
-
-
-	Av vendor.
-
+		- McAfee-SECURE.xml
 
 	Insecure cookies are set for these domains and hosts: ᶜ
-
 		- .mcafee.com
 		- kc.mcafee.com
 		- partnerfocus.mcafee.com
-
 	ᶜ See https://owasp.org/index.php/SecureFlag
 
+	Redirect to http:
+		- www  *
+		- beta *
+		- secure
+	* Partial https
+
+	Dismatch:
+		- cbsphishingquiz
+		- download.beta
+		- downloadcenter
+		- strongmail
+		- uat.planet
+
+	Incomplete certificate chain:
+		- www.blogs
+
+	Timeout:
+		- apps-community
+		- partner
+		- phishingquiz
+		- att-dr
+		- att-prod
+		- exapp
+		- exmobile
+		- exmobiledev
+		- messagingwebservice
+		- notify
+		- service-dr
+		- truekey-dr
+		- truekey-prod
+		- util
+		- verizon-dr
+		- verizon-prod
+
+	Null response:
+		- ngfwlicenses
+	
+	403:
+		- app
+		- service-home
+
+	Reset:
+		- autodiscover
+		- jobs
+		- owa
+
+	Refused:
+		- cp
+		- podcasts
+
 -->
 <ruleset name="McAfee.com (partial)">
-
 	<target host="mcafee.com" />
+	<target host="www.mcafee.com" />
+		<!--	Redirects to http	-->
+		<exclusion pattern="^http://(www\.)?mcafee\.com/($|\w+/index\.html)" />
+		<exclusion pattern="^http://(www\.)?mcafee\.com/(?!whitelisting|apps/about/website-feedback).*\.aspx" />
+		<rule from="^http://(www\.)?mcafee\.com/"
+			to="https://www.mcafee.com/" />
+			<!--	Excluded	-->
+			<test url="http://mcafee.com/" />
+			<test url="http://www.mcafee.com/" />
+			<test url="http://mcafee.com/en/index.html" />
+			<test url="http://www.mcafee.com/en/index.html" />
+
+			<test url="http://mcafee.com/us/partners.aspx" />
+			<test url="http://www.mcafee.com/us/partners.aspx" />
+			<test url="http://mcafee.com/us/business-home.aspx" />
+			<test url="http://www.mcafee.com/us/business-home.aspx" />
+			<test url="http://www.mcafee.com/us/downloads/free-tools/getsusp.aspx" />
+			<test url="http://www.mcafee.com/apps/downloads/security-updates/security-updates.aspx" />
+			<!--	Rewritten	-->
+			<test url="http://www.mcafee.com/img/logo-intel-security.png" />
+			<test url="http://www.mcafee.com/static/css/screen_small.css" />
+			<test url="http://www.mcafee.com/whitelisting/pages/home.aspx" />
+			<test url="http://www.mcafee.com/apps/about/website-feedback.aspx?region=us" />
+
+	<target host="agent.mcafee.com" />
+	<target host="apps-planet.mcafee.com" />
+	<target host="att.mcafee.com" />
+	<target host="att-beta.mcafee.com" />
+
 	<target host="beta.mcafee.com" />
+		<exclusion pattern="^http://beta\.mcafee\.com/(?![Bb][Ee][Tt][Aa][Mm][Cc][Aa][Ff][Ee][Ee]/\S).*" />
```

Comment:
urls are handled case insensitive anyway.
It's also not clear for me what you try to exclude here or why this exclusion is needed.
I think the `\S` and `.*` excludes more than necessary.